script/verify-go-modules.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
```

Comment:
Is python3 available by default on Mac and Windows though?

Reply:
I don't have Windows machines nearby. Regarding macOS, Python 3 is coming from Command Line Developer Tools, which provide `make`, `git` and other tools. So it would be okay assuming people have it.

Reply:
Windows doesn't pre-install Python. You can use one click to install from Microsoft Store.  [Ref](https://devblogs.microsoft.com/python/python-in-the-windows-10-may-2019-update/)

Reply:
Yea, they stubbed python.exe on new installs to open the store to download the community edition. So if we're going for "something that's on every box by default" this isn't it, but it's definitely easy enough to get